### DISCUSSION TITLE
Permissions to create a pull request

### DISCUSSION THREAD

**Discussion by edwinzrodriguez** [Ericsson/codechecker#4002] [Q&A] [answered]
What is the expected procedure to make pull requests?  Do I need to request permissions first and how would I do that?

`
ERROR: Permission to Ericsson/codechecker.git denied to edwinzrodriguez.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.`

**Comment by bruntib** [comment, marked as answer]
Hi,
You should fork Ericsson/codechecker first. You can create pull requests from your own fork to Ericsson/codechecker.
